packaging/freedesktop/alvr.desktop

```
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=ALVR
+GenericName=Game
+Comment=Stream VR games from your PC to your headset via Wi-Fi
+Exec=/usr/bin/alvr_launcher
```

```suggestion
Exec=alvr_launcher
```
You can probably not specify the full path to avoid assuming FHS.